base_location_nuts/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'NUTS Regions',
+    'category': 'Localisation/Europe',
+    'version': '11.0.1.0.0',
+    'depends': [
+        'sales_team',
```

Comment:
I think that you can now remove this dependency. It was required for the menuitem.